docker_explorer/lib/container.py

```diff
@@ -248,7 +248,13 @@ def Mount(self, mount_dir):
       print(c)
     print('Do you want to mount this container ID: {0:s} on {1:s} ?\n'
           '(ie: run these commands) [Y/n]'.format(self.container_id, mount_dir))
-    choice = raw_input().lower()
+    # Ugly Py2/Py3 compat code.
+    # Undo in 2020+
+    try:
+      input = raw_input  # pylint: disable=redefined-builtin
```

Comment:
I would do the built-in redefinition at the top of the file so you can reuse it elsewhere in the file and it doesn't pollute the function's logic